aws/athena.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "How to create an AWS Athena data service with Pulumi"
+cloud: "AWS"
+service: "Athena"
+description: "is an interactive query service that makes it easy to analyze data in Amazon S3 using standard SQL"
+library: "@pulumi/aws"
+brand: "aws"
+og:
+    description: "Use Pulumi to code, deploy, and manage cloud, serverless, and container apps and infrastructure"
+    image: "/images/service/aws-athena.png"
+---
+<!-- Links -->
+[AWS here]: https://aws.amazon.com/athena/
+[Reference docs]: ../reference/aws.html
+[Example code]: https://github.com/pulumi/examples/tree/master/aws-js-s3-folder 
```

Comment:
This link isn't used and links to an s3 example. Applies to some of the other services as well.